embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/network/logging/EmbraceNetworkLoggingService.kt

```diff
@@ -65,103 +43,33 @@ internal class EmbraceNetworkLoggingService(
     }
 
     /**
-     * Process network calls to assert that no limits are exceeded.
+     * Records network calls as spans if their domain can be parsed and is within the limits.
      *
-     * @param networkRequest the network request to process
+     * @param networkRequest the network request to record
      */
-    private fun processNetworkRequest(networkRequest: EmbraceNetworkRequest) {
+    private fun recordNetworkRequest(networkRequest: EmbraceNetworkRequest) {
+        // TODO: Shouldn't we ignore domains that can't be parsed for networkCapturedData too?
```

Comment:
Log a KTLO task and move on - not sure how these are related, so lets detangle this later